### PR TITLE
Make BMO as part of CAPM3 deployment

### PR DESCRIPTION
**Pull request by furkatgofurov7**
This PR deploys BMO as part of capm3-controller manager.
This PR depends on [PR#68](https://github.com/metal3-io/cluster-api-provider-metal3/pull/68)

### CODE REVIEW

File: deploy/bmo-capm3/kustomization.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- ../rbac
+- ../crds
+- ../operator/no_ironic
+configMapGenerator:
+- envs:
+  - ironic_bmo_configmap.env
+  name: ironic-bmo-configmap
```
Comment (Xenwar) [inline deploy/bmo-capm3/kustomization.yaml:10]: Is this indentation correct ?

Comment (Xenwar) [reply deploy/bmo-capm3/kustomization.yaml:10]: I am referring to line number 10.

Comment (maelk) [reply deploy/bmo-capm3/kustomization.yaml:10]: looks correct to me

**Comment by maelk** [comment]
maybe add a small readme in that folder to say what it is used for ?

File: deploy/bmo-capm3/kustomization.yaml (hunk 1 above)
Comment (furkatgofurov7) [reply deploy/bmo-capm3/kustomization.yaml:10]: Yes, it should be

**Comment by furkatgofurov7** [comment]
> maybe add a small readme in that folder to say what it is used for ?
Done

**Comment by kashifest** [comment]
You have to remove WIP from title, if this PR is ready to go in.

**Pull request merged**